docs/competition/team-support/battery-charging.md

```diff
@@ -7,44 +7,76 @@ original:
 
 The batteries themselves have many built in safety features and, unless severely abused, are completely safe. All batteries undergoing charging must be placed inside a fireproof charging bag. In the extremely unlikely event of a fire due to a battery DO NOT attempt to tackle the fire - the batteries contain a flammable metal (Lithium) which cannot be safely tackled with most fire extinguishers.
 
-If at any point you have any doubts about safety, immediately summon the Competition Team Support Coordinator (Rich Barlow) and the Health & Safety Coordinator (Scarzy) via [Helpdesk](/competition/team-support/helpdesk).
+If at any point you have any doubts about safety, immediately ask the helpdesk to summon the Health & Safety officer.
 
 ## Procedures
 
-### Swapping a team's battery
+### Swapping a team's battery - Helpdesk
+
+- Teams will bring their discharged battery to the Helpdesk
+- Discharged batteries should be placed in the discharged battery box
+    - If the battery is damaged, red tape should be placed on it and it should be placed in the damaged LiPo bag instead of the discharged battery box. 
+- A charged battery should be retrieved from the charged battery box and handed to the team.
+    - Do not allow teams to take batteries from the box themselves.
+    - If multiple batteries are returned, only give out one charged battery to each team.
+    - In the event that no charged batteries are in the box, contact the battery charging station over the radio to request more.
+
+### Swapping batteries - Shuffling
+
+The volunteer that is running the battery charging station will periodically have to go and get discharge batteries from the helpdesk, simultaneously dropping off charged batteries. The helpdesk may also contact the volunteer to do it sooner.
+
+- Charged LiPos should be taken to the helpdesk.
+- Discharged and damaged LiPos should be taken to the battery charging station.
+- Any batteries that were marked as damaged by the helpdesk, should be inspected and placed in the damaged battery box if they are damaged.
 
-Teams will bring discharged batteries to the Battery Charging station and require a replacement charged battery. A Battery Charging volunteer should receive the discharged battery from the team and place it in the discharged battery box. A charged battery should be removed from the charged battery box and handed to the team. Do not allow teams to take batteries from the box themselves - we want to prevent teams from hoarding batteries.
+### Charging a discharged battery - Battery Charging Station
 
-### Charging a discharged battery
+All of the battery chargers will be set-up and ready to use. Before a battery is connected the screen must display `LiPo BALANCE 2.2A 11.1V(3S)`. If it displays anything else then please do not use it and ask Helpdesk to summon help.
```

Comment:
Who (which role) should helpdesk summon at this point?

Reply:
Which role should be summoned here? There was nothing previously written about which role should be summoned.

Reply:
Again I think this should go to the person who is overall responsible for the battery charging and likely the person who did the training for the role.

Apologies, my question here was meant to be identifying that we should specify who to ask for help -- we need to ensure that there aren't any dead ends in the instructions we're providing to ensure that things are clear on the day.